src/map/script.c

```diff
@@ -12618,6 +12622,47 @@ BUILDIN(waitingroomkickall) {
 	return true;
 }
 
+/// Kicks target player from the waiting room of the current or target npc.
+///
+/// kickwaitingroom "<npc_name>"{,"<name>"|<account id>};
+/// kickwaitingroom;
+BUILDIN(waitingroomkick) {
+	struct npc_data* nd;
```

Comment:
`struct npc_data *nd;`
see the [coding guidelines chapter 3.1](https://github.com/HerculesWS/Hercules/wiki/Coding-Style#31--spaces)

>When declaring pointer data or a function that returns a pointer type, the preferred use of * is adjacent to the data name or function name and not adjacent to the type name. Examples:

	char *linux_banner;
	unsigned long long memparse(char *ptr, char **retptr);
	char *match_strdup(substring_t *s);